apps/genetics/src/pages/StudyLocusPage/StudyLocusPage.jsx

```diff
@@ -134,7 +142,7 @@ const tableColumns = [
     id: 'gene.symbol',
     label: 'Gene',
     comparator: (a, b) => d3.ascending(a.gene.symbol, b.gene.symbol),
```

Comment:
Apply the same `d3 import` refactor commented  previously

Reply:
Done